tmt/cli.py

```diff
@@ -268,6 +274,9 @@ def run(context, id_, **kwargs):
 @click.option(
     '-c', '--condition', 'conditions', metavar="EXPR", multiple=True,
     help="Use arbitrary Python expression for filtering.")
+@click.option(
+    '--link', 'links', metavar="REL_LINK", multiple=True,
```

Comment:
```suggestion
    '--link', 'links', metavar="RELATION:TARGET", multiple=True,
```
Maybe, we could directly suggest the relation-target format in the metavar instead of the comment?